src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"    method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
```

Comment:
Fix your formatting EVERYWHERE according to [checklist's](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd point
```suggestion
          <input
            type="text"
            name="Surname"
            autocomplete="off"
          >
```